src/helpers/fromHex.js

```diff
@@ -0,0 +1,28 @@
+import { toUtf8, toDecimal } from 'web3-utils'
+export const fromHexToUtf8 = () =>
+  /**
+   * Returns the UTF-8 string representation of a given hex value
+   *
+   * @param {string} hex The hex string
+   * @return {radspec/evaluator/TypedValue}
+   */
+  hex => {
```

Comment:
I'd recommend keeping this as just one helper, `(hex, toDecimal = false) => `, which can then be used as `@fromHex(bytes, true)` for decimal or `@fromHex(bytes)` for UTF8. Alternatively we could pass the mode (utf8 or dec) as a string

Reply:
Think it would make most sense to pass a string, with 'ascii', 'number', and 'utf8' as available options.

Reply:
Done.